cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -208,9 +212,9 @@ func (m command) validateFlags(c *cli.Context, config *config) error {
 		config.cdi.enabled = false
 	}
 
-	if config.runtimeConfigOverrideJSON != "" && config.runtime != "containerd" {
-		m.logger.Warningf("Ignoring runtime-config-override flag for %v", config.runtime)
-		config.runtimeConfigOverrideJSON = ""
+	if config.executablePath != "" && config.runtime != "containerd" && config.runtime != "crio" {
```

Comment:
containerd, crio and docker are the only recognised runtimes by the code base. So wouldn't it just suffice to check if the runtime is docker instead?

Reply:
Docker is also supported by the `nvidia-ctk runtime configure` command.

Reply:
Updated.